crates/turbopack-ecmascript-plugins/src/transform/relay.rs

```diff
@@ -0,0 +1,65 @@
+use std::path::PathBuf;
+
+use async_trait::async_trait;
+use swc_core::{
+    common::{util::take::Take, FileName},
+    ecma::{
+        ast::{Module, Program},
+        visit::FoldWith,
+    },
+};
+use turbo_tasks_fs::FileSystem;
+use turbopack_ecmascript::{CustomTransformer, TransformContext};
+
+#[derive(Debug)]
+pub struct RelayTransformer {
```

Comment:
Decide to move this transform into turbopack side, as it is possible turbopack standalone might need this transform as well.